src/main/java/page/objects/LogInPage.java

```diff
@@ -12,4 +13,15 @@ public void clickCreateAccountButton() {
         click(By.cssSelector(".action.create.primary"));
     }
 
+    public void inputValidUserName() {
+        setText(By.cssSelector("#email"), Constants.VALID_UESR_NAME);
+    }
+
+    public void inputPassword() {
+        setText(By.cssSelector("#pass"), Constants.PASSWORD);
+    }
+
+    public void clickSignInButton() {
+        click(By.cssSelector(".fieldset.login .actions-toolbar .primary .action.login.primary"));
+    }
```

Comment:
CSS locator is too long. Please find a way to make it shorter

Reply:
done